apps/voucherify/src/components/locations/DialogView.jsx

```diff
@@ -0,0 +1,107 @@
+import React, { useEffect, useState } from 'react';
+import { useAutoResizer, useSDK } from '@contentful/react-apps-toolkit';
+import { Button, Modal, Flex, Radio, Pagination, Text, TextLink } from '@contentful/f36-components';
+import { fetchUnlinkedResources } from '../../api/fetchResources';
+import { linkEntryToResource } from '../../api/updateResource';
+import SearchForm from '../SearchForm';
+import { ITEMS_PER_PAGE } from '../../contentful-constants';
+import { ExternalLinkIcon } from '@contentful/f36-icons';
+
+const DialogView = () => {
+  const sdk = useSDK();
+  const { appId, secretKey, customURL } = sdk.parameters.installation.credentials;
+  const [selectedResourceId, setSelectedResourceId] = useState();
```

Comment:
Just a small comment here, when I was toggling between radio buttons in the dialog, I saw the React error `A component is changing an uncontrolled input to be controlled`. If you set this state as an empty string, that should fix it 😄 

```suggestion
  const [selectedResourceId, setSelectedResourceId] = useState('');
```